src/kits/tracker/ContainerWindow.cpp

```diff
@@ -563,6 +525,29 @@ BContainerWindow::BContainerWindow(LockingList<BWindow>* list,
 		list->AddItem(this);
 	}
 
+	if (useLayouts) {
+		SetFlags(Flags() | B_AUTO_UPDATE_SIZE_LIMITS);
+
+		fRootLayout = new BGroupLayout(B_VERTICAL, 0);
+		fRootLayout->SetInsets(0);
+		SetLayout(fRootLayout);
+		fRootLayout->Owner()->SetViewColor(ui_color(B_PANEL_BACKGROUND_COLOR));
+
+		if (!fIsDesktop) {
+			fMenuContainer = new BGroupView(B_HORIZONTAL, 0);
+			fRootLayout->AddView(fMenuContainer);
+
+			fPoseContainer = new BGridView(0.0, 0.0);
+			fRootLayout->AddView(fPoseContainer);
+
+			fBorderedView = new BorderedView;
+			fPoseContainer->GridLayout()->AddView(fBorderedView, 0, 1);
```

Comment:
The BColumnListView in the Open with... window doesn't seem to be resizing correctly, see: http://postimg.org/image/8aq8tlvcl/ (while the link lasts)